internal/dlog/calc.go

```diff
@@ -219,15 +290,16 @@ func (c *CalcBN256) precompute(g *bn256.GT) {
 	c.Precomp = T
 }
 
-// BabyStepGiantStepBN256 implements the baby-step giant-step method to
+// BabyStepGiantStepStand implements the baby-step giant-step method to
 // compute the discrete logarithm in the BN256.GT group.
 //
-// It searches for a solution <= sqrt(bound). If bound argument is nil,
-// the bound is automatically set to p-1.
+// It searches for a solution <= bound. If bound argument is nil,
+// the bound is automatically set to the hard coded MaxBound.
 //
-// The function returns x, where h = g^x mod p. If the solution was not found
+// The function returns x, where h = g^x in BN256.GT group where operations
+// are written as multiplications. If the solution was not found
 // within the provided bound, it returns an error.
-func (c *CalcBN256) BabyStepGiantStep(h, g *bn256.GT) (*big.Int, error) {
+func (c *CalcBN256) BabyStepGiantStepStand(h, g *bn256.GT) (*big.Int, error) {
```

Comment:
What's the reason for the name suffix `Stand`?

Reply:
I considered it as the "standard" version of the baby-step giant-step algorithm. Any better suggestion?

Reply:
A shorter version might be `Std`.

Reply:
Ok.